crates/bevy_ecs/src/storage/mod.rs

```diff
@@ -1,4 +1,24 @@
 //! Storage layouts for ECS data.
+//!
+//! This module contains the low level backing data stores for ECS. These all offer minimal and often
```

Comment:
What is a “low level backing datastore”? What does low-level mean in this context?

Reply:
Low level meaning it's meant to be the base abstraction above the language primitives. We're directly working with std/alloc/core APIs here.

Backing datastore means it's the authoritative "place" within the ECS where all of the memory for the World is allocated. It's what "backs" the rest of the user-facing interface.

Reply:
These might be good comments to throw in as well! Makes sense